package.json

```diff
@@ -28,8 +28,9 @@
   "system": {},
   "dependencies": {
     "escape-html": "^1.0.3",
-    "unescape-html": "^1.0.0",
-    "jquery": "^2.2.4"
+    "jquery": "^2.2.4",
+    "lodash": "^4.17.4",
```

Comment:
You could install https://www.npmjs.com/package/lodash.debounce instead to save some install time (17kb instead of 1.4mb for all of lodash).